acct-mgt-test.py

```diff
@@ -101,7 +101,11 @@ def user(user_name, op, success_pattern, auth_opts=[]):
     #    stderr=subprocess.STDOUT,
     # )
     cmd = ["curl", "-X", op, "-kv"] + auth_opts[url + "/users/" + user_name]
-    result = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,)
+    result = subprocess.run(
+        cmd,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+    )
```

Comment:
What does this function do? I don't see it being called, and it doesn't return a result.